dist-scripts/smoke-test.sh

```diff
@@ -1,4 +1,4 @@
-http=$(curl -s -o /dev/null -w "%{http_code}" -i -X GET  -H "Content-Type: application/json" -d '{"candidateId": "someCandidateId","faults":[{"id":"juncSpeed","faultsNo":10,"isSerious":false,"isDangerous":false}]}' "${JOURNAL_API_URL}")
+http=$(curl -s -o /dev/null -w "%{http_code}" -i -X GET  -H "Content-Type: application/json" "${JOURNAL_API_URL}")
```

Comment:
is it working? when I try to get for just base URL I get 503

Reply:
Globals JOURNAL_API_URL has been updated with correct url.